axicfg/vector.toml

```diff
@@ -0,0 +1,18 @@
+[sources.src]
```

Comment:
Was this intended to get committed?

Reply:
Oooops, that's what you get for having a `git commit --all` alias in muscle memory and not checking changes on rebase. Invalidated all tokens and removed the file from history 🙈